BookingProject/BookingWebAPI/AuditWebApi.Infrastructure/Configuration/KafkaSettings.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AuditWebApi.Infrastructure.Configuration
+{
+    public class KafkaSettings
+    {
+        public string BootstrapServers { get; set; }
+        public string Topic { get; set; }
```

Comment:
Configure topics in the application. U will publish to multiple topics and u cannot configure them via appsettings. I suggest creating `Topics` class which will be responsible for topic description (names, amount of partitions, replica factor, etc)